.gitignore

```diff
@@ -12,6 +12,7 @@
 *.iws
 *.org
 .project
+pmip
```

Comment:
My IDE started adding those files to a project, which is annoying.  So I added that file format to the `.gitignore` for that reason.